examples/atlas/log-download.md

```diff
@@ -0,0 +1,9 @@
+# Download and Unzip Logs
+
+You can download and unzip logs in one go like
```

Comment:
One does not "unzip" a gzip-compressed file. One could "gunzip" or "decompress" it.